packages/cli-template-monorepo-ethers/apps/web-app/src/app/globals.css

```diff
@@ -33,18 +44,30 @@ body {
 
 p {
     line-height: 1.5rem;
+    font-weight: 300;
+    overflow-wrap: break-word;
+    font-size: 1rem;
+}
+
+b {
+    font-weight: 600;
+}
+
+.key-wrapper {
+    padding-bottom: 1.5rem;
+    padding-left: 0.5rem;
 }
 
 .container {
     display: flex;
     flex-direction: column;
     height: 100%;
-    max-width: 32rem;
-
+    max-width: 40vw;
+    min-width: 35rem;
     margin: auto;
     padding: 1rem;
-
-    min-height: calc(100vh - 3.5rem);
```

Comment:
I think it's worth keeping `min-height: calc(100vh - 3.5rem);` to have a better UI.